sound_manager.py

```diff
@@ -0,0 +1,64 @@
+from arcade import Sound, load_sound
+
+DEFAULT_VOLUME = 0.7
+SFXPATH_BEETLE_HIT = "Assets/Sound/SFX/impact2_noisecollector.wav"
+SFXPATH_BEETLE_DEAD = "Assets/Sound/SFX/beetle_dead_mixkit.wav"
+SFX_PATH_PEASHOOTER = "Assets\Sound\SFX\pew_xtrgamr.wav"
```

Comment:
Make sure your naming is consistent here. In fact, I would consider just naming these like `BEETLE_HIT_PATH` since `SFX` is already clear from the context.

Also make sure to use slashes `/` not backslashes `\` because to use backslashes, you have to escape them with `\\`.

Reply:
Yeah good point, this is a holdover from when they were in different docs. I'll redo these similar to how you have it in the review commen.